### DISCUSSION TITLE
MIOTA token from Firefly wallet not transferable to Binance

### DISCUSSION THREAD

**Discussion by WoodwayJoe** [iotaledger/firefly#8294] [Help & Support]
Hallo,

ich kann keine IOTA von meiner Firefly Wallet auf meinen Binance Account senden.

Ich habe mehrfach einen DEPOSIT Link über Binance erzeugt und versucht verschiedene Mengen IOTA zu transferieren (1.000 IOTA, 10 IOTA)

Zur Sicherheit habe ich ein Ablaufdatum gesetzt (mal einen Tag, mal eine Woche)

genutzte Firefly Version: 2.0.9
genutzte Windows 11 Version: 23H2 (Build:22631.3296)

Kennt jemand das Problem?
Wie kann ich meine IOTA zu Binance transferieren?
Oder kann ich meine IOTA auch über die Firefly Wallet verkaufen?

Ich komme aus Deutschland und habe den Text übersetzen lassen.

Vielen Dank für eure Rückmeldung und viele Grüße.
Stefan

**Comment by WoodwayJoe** [comment]
Hello,

I can't send IOTA from my Firefly wallet to my Binance account.

I have created a DEPOSIT link via Binance several times and tried to transfer different amounts of IOTA (1,000 IOTA, 10 IOTA)

To be on the safe side, I set an expiration date (sometimes one day, sometimes one week)

Firefly version used: 2.0.9
Windows 11 version used: 23H2 (Build:22631.3296)

Does anyone know the problem?
How can I transfer my IOTA to Binance?
Or can I also sell my IOTA via the Firefly wallet?

I am from Germany and have had the text translated.

Thank you very much for your feedback and best regards.
Stefan

Translated with DeepL.com (free version)

**Reply by begonaalvarezd** [reply]
Exchanges dont allow setting expiration times because that needs claiming on their end and they dont do it, so your ttransaction will just expire. You have to send without any special unlock conditio, that means, without expiration

**Reply by WoodwayJoe** [reply]
Thank you very much for the quick reply. That's how it worked.

Best regards
